exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -160,6 +160,9 @@
   OptionValidator OUTPUT_FORMAT_VALIDATOR = new StringValidator(OUTPUT_FORMAT_OPTION, "parquet");
   String PARQUET_BLOCK_SIZE = "store.parquet.block-size";
   OptionValidator PARQUET_BLOCK_SIZE_VALIDATOR = new LongValidator(PARQUET_BLOCK_SIZE, 512*1024*1024);
+  String PARQUET_WRITER_USE_CONFIGURED_BLOCK_SIZE = "store.parquet.writer.use_configured_block-size";
```

Comment:
Making my contribution to the review thread:
`match_filesystem_block-size`
The value would be the logical inverse of the original flag.